oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -922,12 +977,19 @@ void pldm::responder::oem_ibm_platform::Handler::buildOEMPDR(
     attachOemEntityToEntityAssociationPDR(
         this, bmcEntityTree, "/xyz/openbmc_project/inventory/system", repo,
         fwUpEntity);
+    pldm_entity saiEntity = {PLDM_OEM_IBM_ENTITY_REAL_SAI, 1, 1};
+    attachOemEntityToEntityAssociationPDR(
+        this, bmcEntityTree, "/xyz/openbmc_project/inventory/system", repo,
+        saiEntity);
     pldm_entity powerStateEntity = {PLDM_OEM_IBM_CHASSIS_POWER_CONTROLLER, 0,
                                     1};
     attachOemEntityToEntityAssociationPDR(
         this, bmcEntityTree, "/xyz/openbmc_project/inventory/system", repo,
         powerStateEntity);
 
+    realSAISensorId = findStateSensorId(
+        repo.getPdr(), 0, PLDM_OEM_IBM_ENTITY_REAL_SAI, ENTITY_INSTANCE_1, 1,
+        PLDM_STATE_SET_OPERATIONAL_FAULT_STATUS);
```

Comment:
this can be moved out from this function and done in `processSaiUpdate` function itself? that way we don't want a private class variable `realSAISensorId` ?

Reply:
repo is passed from void pldm::responder::oem_ibm_platform::Handler::buildOEMPDR() and not present under class Handler